tests/v2/actions/vai/vai.go

```diff
@@ -0,0 +1,21 @@
+package vai
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/rancher/shepherd/clients/rancher"
+	"github.com/rancher/shepherd/extensions/vai"
+)
+
+func EnableVAI(client *rancher.Client, vaiEnabled *bool, once *sync.Once) error {
+	var enableErr error
+	once.Do(func() {
```

Comment:
this is just for my own understanding but why do we need the once.Do?

Reply:
this is to make sure that enabling happens only once. especially useful in concurrent runs that makes sure even if the function is called multiple times, its executed only once. its not important for this particular scenario but for the tests in vai_test.go its important that this is defined.

Reply:
I removed the changes made in actions for vai and instead used extensions directly.